config.example.toml

```diff
@@ -603,6 +603,13 @@ change-id = 116881
 # desired in distributions, for example.
 #rpath = true
 
+# Indicates whether `rustc` should be stripped of symbols using `-Cstrip=symbols`.
+#strip = false
+
+# Indicates whether `rustc` should be compiled with stack protectors.
```

Comment:
Both of these comments say "rustc" but this is applying to all Rust binary (i.e. not source code) outputs -- e.g., cargo, libstd as well. Can we adjust the comment to avoid mentioning rustc specifically?

Reply:
Modified according to comments.